step.yml

```diff
@@ -90,53 +90,73 @@ inputs:
+        - `apple_id`: Use Apple Developer Connection and Application-specific password Step input. Only Apple ID (legacy) authentication is used. Other authentication related Step inputs are ignored.
+        - `off`: Do not use Apple Developer Connection. Only authentication related Step inputs are considered.
+      is_required: true
+      value_options:
+      - "automatic"
+      - "api_key"
+      - "apple_id"
+      - "off"
+  - api_key_path: ""
+    opts:
+      title: "API Key: URL"
+      summary: Path to local or remote file
+      description: |-
+        Specify the path in an URL format where your API key is stored.
+        For example: `https://URL/TO/AuthKey_something.p8` or `file:///PATH/TO/AuthKey_something.p8`
+        You can upload your key on the **Generic File Storage** tab in the Workflow Editor and set the environment variable for the file here.
+        For example: `$BITRISEIO_MYKEY_URL`
+        **NOTE:** Only available from XCode 11.0 and above.
+      is_sensitive: true
+  - api_issuer: ""
+    opts:
+      title: "API Key: Issuer ID"
+      description: |-
+        Issuer ID. Required if API Key URL (api_key_path) is specified.
+        **NOTE:** Only available from XCode 11.0 and above.
   - itunescon_user: ""
     opts:
-      title: "Apple ID"
+      title: "Apple ID (legacy): Apple ID"
       description: Login Apple ID for iTunes Connect.
       is_required: false
       is_sensitive: true
   - password: ""
     opts:
-      title: "Password"
+      title: "Apple ID (legacy): Password"
       description: Password for the specified Apple ID.
       is_required: false
       is_sensitive: true
   - app_password: ""
     opts:
-      title: "Application-specific password"
+      title: "Apple ID (legacy): Application-specific password (Required if using legacy Bitrise Apple Developer Connection)"
       description: |-
         An application-specific password for the Apple ID.
         **NOTE:** Application-specific passwords can be created on the
         [AppleID Website](https://appleid.apple.com). It can be used to
         bypass two-factor authentication.
       is_required: false
       is_sensitive: true
-  - api_key_path: ""
-    opts:
-      title: "API Key URL"
-      description: |-
-        Specify the path in an URL format where your API key is stored.
-        For example: `https://URL/TO/AuthKey_something.p8` or `file:///PATH/TO/AuthKey_something.p8`
-        You can upload your key on the **Generic File Storage** tab in the Workflow Editor and set the environment variable for the file here.
-        For example: `$BITRISEIO_MYKEY_URL`
-        **NOTE:** Only available from XCode 11.0 and above.
-  - api_issuer: ""
-    opts:
-      title: "Issuer ID"
-      description: |-
-        Issuer ID. Required if api_key_path is specified.
-        **NOTE:** Only available from XCode 11.0 and above.
   - team_id: ""
     opts:
-      title: "Team ID"
+      title: "Apple ID (legacy): Team ID"
```

Comment:
Please do not mention legacy, for now, it can be frustrating to use an input with this title, and might some user can not use API key authentication

Reply:
fixed